src/paths/distances.c

```diff
@@ -777,3 +777,50 @@ igraph_error_t igraph_pseudo_diameter_dijkstra(const igraph_t *graph,
 
     return IGRAPH_SUCCESS;
 }
+
+/**
+ * \function igraph_graph_center
+ * \brief Centre vertices of a graph.
+ *
+ * The centre vertices of a graph is calculated by finding the vertices
```

Comment:
```suggestion
 * The central vertices of a graph is calculated by finding the vertices
```

Reply:
These are stylistic suggestions, but something of note is to use American spelling, as the rest of the documentation does. I also prefer British spelling in general, but I use American spelling for igraph documentation.